### ISSUE TITLE
[BUG] bottom pause/play, slider etc controls not longer responding after video started on IOS

### ISSUE THREAD

**Issue by Harmenmeijer** [sampotts/plyr#672]
- [x] Issue does not already exist
- [x] Issue observed on https://plyr.io

### Expected behaviour 
On iPad, after a video has been started playing using the large center play button the bottom controls should become visible after a tap anywhere on the video and the user must be able to interact with it. After 3 seconds of inactivity the bottom controls thould be hidden again. 

Secondly: After having started the video using the bottom play button the bottom controls should hide after a few seconds

### Actual behaviour
While a video is playing a tap on the video brings the bottom controls in view but they disappear immediately thereafter. (no 3 seconds wait) It is not possible to interact with the slider or the pause button. 
secondly: after starting the video using the bottom play button the controls never hide.

### Environment

- Browser: Safari or Chrome
- Version:  
- Operating System: IOS 11
- Version: IOS 11 on iPad

Players affected:
- [x] HTML5 Video
- [not tested] HTML5 Audio
- [not tested ] YouTube
- [ not tested] Vimeo

### Steps to reproduce 
- open Safari and goto https://plyr.io
- start video using the large central play button
- tap anywhere on the video area to bring up the bottom controls.
- try to pause the video or user the slider. This won't work as the controls are immediately hidden again.

I've found a partial fix. Alter the code at line 2625 to this:
`            // If the mouse is not over the controls, set a timeout to hide them
            if (show || plyr.media.paused || loading) {
                _toggleClass(plyr.container, config.classes.hideControls, false);

                // Always show controls when paused or if touch
                if (plyr.media.paused || loading) {
                    return;
                }
                // this is where the delay used to be.
            }

           // Delay for hiding on touch <-- moved here 
           if (plyr.browser.isTouch) {
               delay = 3000;
           }
`
Basically I've moved the delay outside of the 'if' statement at line 2625. This activates the delay at all times on touch devices. This makes it possible for a user to interact with the bottom controls within 3 seconds.

**Comment by sampotts** [comment]
Can you try this again in v3? I have a feeling this should be resolved but if not, feel free to re-open 👍
